### DISCUSSION TITLE
My account has been blocked

### DISCUSSION THREAD

**Discussion by VulcanWM** [vercel/community#1879] [Help]
### Summary

My account has been recently blocked.
I did not know that I had reached the limit until I saw the blocked tag on my username.
The limit has only been exceeded because of this bug I have been hitting in my project (guest-book) for a few hours.
It seems to be working fine when I run it locally, but it doesn't seem to work when I host on Vercel.
Because of this, I kept on creating new commits to make it work but it kept on failing and using a lot of the limit.
I can assure you that I am creating a small-scale project that won't use much data.

**Comment by amyegan** [comment]
Hi @VulcanWM. You should have received an email notification explaining why you were blocked. The email outlines the next steps you must take to get your account unblocked (ref. [Why has my account or deployment been blocked?](https://vercel.com/guides/why-is-my-account-deployment-blocked))
